rest/route/github.go

```diff
@@ -29,11 +29,38 @@ const (
 	githubActionSynchronize = "synchronize"
 	githubActionReopened    = "reopened"
 
-	retryComment = "evergreen retry"
-	patchComment = "evergreen patch"
-	refTags      = "refs/tags/"
+	// pull request comments
+	retryComment   = "evergreen retry"
+	patchComment   = "evergreen patch"
+	triggerComment = "evergreen merge"
+
+	refTags = "refs/tags/"
 )
 
+func trimComment(comment string) string {
+	return strings.Join(strings.Fields(strings.ToLower(comment)), " ")
+}
+
+func isRetryComment(comment string) bool {
```

Comment:
Can we add a couple of little test cases to each of these that contain white space?